test/distributed/_composable/fsdp/test_fully_shard_memory.py

```diff
@@ -23,11 +27,15 @@ def world_size(self) -> int:
     @skip_if_lt_x_gpu(2)
     def test_fully_shard_training_memory(self):
         self.run_subtests(
-            {"reshard_after_forward": [True, False]},
+            {"reshard_after_forward": [True, False], "use_cpu_offload": [True, False]},
             self._test_fully_shard_training_memory,
         )
 
-    def _test_fully_shard_training_memory(self, reshard_after_forward: bool):
+    def _test_fully_shard_training_memory(
+        self, reshard_after_forward: bool, use_cpu_offload: bool
+    ):
+        if not reshard_after_forward and use_cpu_offload:
+            return  # skip since not a common use case
```

Comment:
Do we not have a utility to note that we skipped the test like `pytest.skip`?

Reply:
I think the problem is that this is using subtests. Do you know if there is such a utility to indicate a subtest is skipped?

(`self.run_subtests` is syntactic sugar for doing the product over configs and running subtests.)